search/tasks.py

```diff
@@ -764,6 +764,8 @@ def start_recreate_index(self, indexes):
                             platform__in=(
                                 PlatformType.ocw.value,
                                 PlatformType.xpro.value,
+                                PlatformType.mitx.value,
+                                PlatformType.mitxonline.value,
```

Comment:
For my own knowledge - do we have/should we have plans to add any other platforms here? Not sure what resources the others have, but I noticed, for example, mm isn't here.  Just curious.

Reply:
Eventually, "Open Learning Library" courses (platform type "oll") will need to be added, once its content archives are available.  These are all edx-based platforms.